tools/scripts/ensure.mjs

```diff
@@ -0,0 +1,46 @@
+import * as fs from 'fs'
+
+console.log(process.argv)
+
+const configuration = {
+  organisation: 'sebspark',
+  package: process.argv[2],
+  path: 'packages',
+}
+
+export default async function ensure(options) {
+  console.log(`Ensuring @${options.organisation}/${options.package} package...`)
+
+  let packageJson = {}
+  const currentDir = process.cwd()
+  const packageJsonPath = `${currentDir}/${options.path}/${options.package}/package.json`
```

Comment:
What are your thoughts on adding a check if the package exists here? Something like:

```js
const packagePath = `${currentDir}/${options.path}/${options.package}`
const packageJsonPath = `${packagePath}/package.json`

if (!fs.existsSync(packagePath)) {
  console.error(
    `ERROR: @${options.organisation}/${options.package} package does not exist.`
  )
  return
}
```

That would display a prettier error if something doesn't exist.

Reply:
Yes that makes sense, good idea.